src/image.tsx

```diff
@@ -60,16 +61,17 @@ type DeprecatedImageProps = {
     aspectRatio?: AspectRatio;
     /** defaults to empty string */
     alt?: string;
-    children?: void;
     dataAttributes?: DataAttributes;
+    children?: void;
+    noBorderRadius?: boolean;
 };
 
 const Image = React.forwardRef<HTMLImageElement, ImageProps | DeprecatedImageProps>(
-    ({aspectRatio = '1:1', alt = '', dataAttributes, ...props}, ref) => {
-        const noBorderRadius = useDisableBorderRadius();
```

Comment:
I think we should still be listening to this context. It's being used in `MediaCard` right now.
Otherwise, in the places where we were using `MediaCards` with `Image` we'll see border radius if we don't explicitly set the new `noBorderRadius` prop.

Reply:
+1

This is the logic:
* default: with border radius
* if the useDisableBorderRadius returns true => remove border radius
* if the prop noBorderRadius is true => remove border radius